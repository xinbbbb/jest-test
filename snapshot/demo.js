export const generateConfig = ()=> ({
  url: 'localhost',
  port: 8080,
  param: "?v=1"
})

export const generateConfigAnother = ()=> ({
  url: 'localhost',
  port: 8081,
  param: "?v=2",
  time: new Date()
})