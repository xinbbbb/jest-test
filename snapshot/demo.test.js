import { generateConfig, generateConfigAnother } from './demo'

test('test snapshot', ()=> {
   expect(generateConfig()).toMatchSnapshot()
})

test('test another snapshot', ()=> {
  expect(generateConfigAnother()).toMatchSnapshot({
    time: expect.any(Date)
  })
})