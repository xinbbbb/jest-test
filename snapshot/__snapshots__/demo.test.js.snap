// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test another snapshot 1`] = `
Object {
  "param": "?v=2",
  "port": 8081,
  "url": "localhost",
}
`;

exports[`test snapshot 1`] = `
Object {
  "param": "?v=1",
  "port": 8080,
  "url": "localhost",
}
`;
